modules/02_welfare/utilitarian/equations.gms

```diff
@@ -58,6 +61,7 @@ $ifthen "%cm_INCONV_PENALTY_FESwitch%" == "on"
 $endif
       )
     )
+
```

Comment:
is the format change (new line) intentional?

Reply:
nope, removed.